.github/workflows/ci-tests.yml

```
@@ -0,0 +1,37 @@
+name: CI Tests
+
+on:
+  pull_request:
+  push:
+    branches:    
+      - master
+
```

```suggestion
  workflow_dispatch:
```
This allows us to run the action ad hoc from the actions tab.